utilities/_layout.scss

```diff
@@ -0,0 +1,32 @@
+//Full responsive layout based on design system
+//Todo: amend spacing calc for between 414px and 599 as some misalignment
+.grid {
+    @include grid(repeat(6, 1fr), 16px, 24px);
+    @media ($mobile-upper <= width <= $desktop-lower) {
+        @include grid(repeat(12, 1fr), 32px, 75px);
+    }
+    @media (width >= ($desktop-lower + 1)) {
+        @include grid(repeat(12, 1fr), 32px, 100px);
+    }
+}
+
+.full-width {
+    grid-column: span 6;
+    @media (width >= $mobile-upper) {
+        grid-column: span 12;
+    }
+}
+
+.col-3 {
+    grid-column: span 6;
+    @media (width >= $mobile-upper) {
+        grid-column: span 4;
+    }
+}
+
+.col-4 {
+    grid-column: span 6;
+    @media (width >= $mobile-upper) {
```

Comment:
Small style point, we'd normally have a blank line above media queries

Reply:
Not a problem, I did have orignally but I think it was one of the errors thrown up by travis, will adjust and run style-lint just in case